roles/keystone/templates/etc/keystone/keystone.conf

```diff
@@ -82,7 +82,6 @@ methods = external,password,token{{ ',oidc' if keystone.federation.sp.oidc.enabl
 [federation]
 driver = keystone.contrib.federation.backends.sql.Federation
```

Comment:
I think they are deprecating this driver: `driver = keystone.contrib.federation.backends.sql.Federation` I think the default `driver = sql` is good enough so think we can remove this line as well.